apps/snapodds-builder/src/index.ts

```diff
@@ -1,3 +1,4 @@
-import { SnapoddsSdkWrapper } from './app/snapodds-sdk-wrapper';
```

Comment:
It doesn't work for me now with the error: Uncaught ReferenceError: SnapoddsSdkWrapper is not defined. I think it is caused by this change.

Reply:
Even when I return this line the snapodds-builder.js (in the dist folder) has the following content:
`window.SnapoddsSdk=SnapoddsSdkWrapper;`

Reply:
fixed